### PR TITLE
Add cc13x2_26x2 build to unified build

### PR DESCRIPTION
**Pull request by mspang**
There are some missing files in build_overrides to build TI in the
unified build, as well as some missing dependencies diagnosed by
gn --check --fail-on-unused-args.

Fix those issues so that cc13x2_26x2 can be built together with the
other examples. Also despam the sysconfig generator so we don't
drown out important diagnostics.

Tested via

  `./gn_build.sh enable_cc13x2_26x2_builds=true ti_simplelink_sdk_root=\"$HOME/ti/simplelink_cc13x2_26x2_sdk_4_40_00_44\" ti_sysconfig_root=\"$HOME/ti/sysconfig_1.7.0\"`

### CODE REVIEW

**Review by saurabhst — APPROVED** [review]

File: third_party/ti_simplelink_sdk/ti_simplelink_sdk.gni [hunk 1]
```diff
@@ -16,8 +16,11 @@ import("//build_overrides/build.gni")
 import("//build_overrides/chip.gni")
 import("//build_overrides/freertos.gni")
 import("//build_overrides/mbedtls.gni")
+import("//build_overrides/openthread.gni")
+import("//build_overrides/pigweed.gni")
 import("//build_overrides/ti_simplelink_sdk.gni")
 
+import("$dir_pw_build/python.gni")
```
Comment (srickardti) [inline third_party/ti_simplelink_sdk/ti_simplelink_sdk.gni:23]: nit: `{ }` around variable

File: third_party/ti_simplelink_sdk/ti_simplelink_sdk.gni [hunk 2]
```diff
@@ -63,7 +66,7 @@ template("ti_sysconfig") {
     }
   }
 
-  action("${target_name}_gen") {
+  pw_python_action("${target_name}_gen") {
```
Comment (srickardti) [inline third_party/ti_simplelink_sdk/ti_simplelink_sdk.gni:69]: what's the advantage of using a pigweed action instead of a gn action here?

**Review by srickardti — APPROVED** [review]

File: third_party/ti_simplelink_sdk/ti_simplelink_sdk.gni (hunk 2 above)
Comment (mspang) [reply third_party/ti_simplelink_sdk/ti_simplelink_sdk.gni:69]: It squelches the output, unless the script fails.

File: third_party/ti_simplelink_sdk/ti_simplelink_sdk.gni (hunk 1 above)
Comment (mspang) [reply third_party/ti_simplelink_sdk/ti_simplelink_sdk.gni]: Done.

**Review by woody-apple — APPROVED** [review]

**Comment by woody-apple** [comment]
@mspang Does this need a rebase post build fix?

**Comment by woody-apple** [comment]
Fixed, @andy31415 ?

File: build_overrides/freertos.gni [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2020 Project CHIP Authors
```
Comment (andy31415) [inline build_overrides/freertos.gni:1]: nit: 2021

**Review by andy31415 — APPROVED** [review]

**Pull request merged**